src/pages/clients/show/Client.tsx

```diff
@@ -76,7 +84,34 @@ export function Client() {
       href: generatePath('/clients/:id/recurring_expenses', { id }),
     },
   ];
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const onSave = (password: string, passwordIsRequired: boolean) => {
+    const toastId = toast.loading(t('processing'));
+    if (password == '') {
+      toast.dismiss();
+      toast.error(t('password_error_incorrect'));
```

Comment:
This is wrong. We need to integrate validation errors into `ValidationBag` interface so we can access errors as if they were from the backend.